### ISSUE TITLE
Emoji markers are not displayed on the map

### ISSUE THREAD

**Issue by link2xt** [deltachat/deltachat-android#1783]
One-letter markers are displayed directly (no need to click on the marker), but emojis don't.

Probably something is wrong with this length check and emojis length is not "1"? I tried replacing it with `neq` and `0`, but then it stopped displaying any markers at all, even 1-letter english markers:
https://github.com/deltachat/deltachat-android/blob/370335bf6e4f28a79a3f0a654035f4dafecb85ad/src/org/thoughtcrime/securesms/map/MapDataManager.java#L459

It's fixed in the core (https://github.com/deltachat/deltachat-android/blob/370335bf6e4f28a79a3f0a654035f4dafecb85ad/src/org/thoughtcrime/securesms/map/MapDataManager.java#L459), and I have tested with python that markers are returned by the FFI just as well as letters.

**Comment by link2xt** [comment]
Maybe the problem is with the fonts, emojis are not rendered even in the black font below the marker:
![image_2021-01-31_01-21-44](https://user-images.githubusercontent.com/18373967/106371966-fc4d8c80-637b-11eb-98f3-e61217a409e3.jpg)

Maybe related issue: https://github.com/nitaliano/react-native-mapbox-gl/issues/1407

**Comment by link2xt** [comment]
I have tried to do something similar to https://docs.mapbox.com/android/maps/examples/text-field-formatting/ by adding:
```
import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.textFont;
...
private static final String[] textFonts = new String[] {"Roboto Black", "Arial Unicode MS Regular", "emojiOne"};
...
                 textAllowOverlap(true),
+                textFont(textFonts),
                 textIgnorePlacement(true)
```
but nothing changed.

Not sure if `emojiOne` is available anywhere or should be uploaded to mapbox account.

**Comment by link2xt** [comment]
As a workaround it should be possible to extract corresponding image from our emoji picker and place them on the map as images.

Or maybe actually replace mapbox with OSM: https://github.com/osmdroid/osmdroid/wiki/How-to-use-the-osmdroid-library

https://support.delta.chat/t/switch-from-mapbox-to-osmdroid-on-android/1124

**Comment by link2xt** [comment]
Here is the commit which added single-character markers for reference: f5cc75723a73a807920083b4e925a16d111554ef

**Comment by cyBerta** [comment]
some links which might show the way to go:
https://github.com/mapbox/mapbox-gl-native/pull/15937
https://github.com/mapbox/mapbox-gl-native-android/pull/324

**Comment by link2xt** [comment]
On Desktop emojis are already displayed correctly with the latest core (master branch).

**Comment by link2xt** [comment]
PR with a fix: #1792
